go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/smartystreets/assertions v0.0.0-20180820201707-7c9eb446e3cf // indirect
 	github.com/smartystreets/goconvey v0.0.0-20180222194500-ef6db91d284a // indirect
 	github.com/stretchr/testify v1.4.0
+	golang.org/x/tools/gopls v0.4.1 // indirect
```

Comment:
why do we need this?

Reply:
Will remove it in newer commits, changing PR to WIP